File: Try_GameEngine5/src/com/example/try_gameengine/framework/IChessPointManager.java
package com.example.try_gameengine.framework;

import java.util.List;

public interface IChessPointManager {
//	IChessPoint createChessPointRed();
//	IChessPoint createChessPointYellow();
//	IChessPoint createChessPointWhite();
	
	List<String> getUseableChessPointList();
	
	IChessPoint createChessPonitRamdon();
}

File: Try_GameEngine5/src/com/example/try_gameengine/framework/IChessPointManager.kt
package com.example.try_gameengine.framework

interface IChessPointManager {
    //	IChessPoint createChessPointRed();
    //	IChessPoint createChessPointYellow();
    //	IChessPoint createChessPointWhite();
    fun getUseableChessPointList(): MutableList<String?>?

    fun createChessPonitRamdon(): IChessPoint?
}
